frontend/src/metabase/query_builder/components/filters/modals/InlineCategoryPicker/InlineCategoryPicker.tsx

```diff
@@ -16,7 +16,6 @@ import { MAX_INLINE_CATEGORIES } from "./constants";
 import { isValidOption } from "./utils";
 
 import { SimpleCategoryFilterPicker } from "./SimpleCategoryFilterPicker";
-import { LargeCategoryFilterPicker } from "./LargeCategoryFilterPicker";
```

Comment:
we don't need this anymore: the popover suggestions in the regular value input do just as good a job